Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -1103,7 +1103,10 @@ def targetName (cfg : Config) (src : Name) : CoreM Name := do
   let res := if cfg.tgt == .anonymous then pre.str tgt_auto else pre1 ++ cfg.tgt
   -- we allow translating to itself if `tgt == src`, which is occasionally useful for `additiveTest`
   if res == src && cfg.tgt != src then
-    throwError "to_additive: can't transport {src} to itself."
+    throwError "to_additive: the generated additivised name equals the original name {src}.\
+    Usually, this means you need to name your declaration correctly \
+    (e.g., name instances with a name that can be additivised). \
+    Sometimes, this means you need to extend to_additive to handle the new names."
```

Comment:
What about
```suggestion
    throwError "to_additive: the generated additivised name equals the original name {src},\
    meaning that no part of the name was additivised.\
    Check that your declaration name is correct (if your declaration is an `instance`, try name it)\
    or provide an additivised name using the 'to_additive my_add_name' syntax."
```

Reply:
I like your suggestion. (I'm adding a test locally, hence tweaking it to make line breaks nice etc.)

Reply:
No, I think it would be _cool_, but not necessarily useful

Reply:
Then let me punt on this in favour of other changes I could make.